shakedown/dcos/package.py

```diff
@@ -173,7 +177,19 @@ def package_installed(package_name, service_name=None):
     """
 
     package_manager = _get_package_manager()
-    return len(package_manager.installed_apps(package_name, service_name)) > 0
+
+    app_installed = len(package_manager.installed_apps(package_name, service_name)) > 0
+
+    subcommand_installed = False
+    for subcmd in package.installed_subcommands():
+        package_json = subcmd.package_json()
+        if package_json['name'] == package_name:
+            subcommand_installed = True
+
+    if (app_installed or subcommand_installed):
```

Comment:
This can be a single return statement itself `return (app installed .....`

Reply:
Updated.